pkgs/build-support/setup-hooks/make-symlinks-relative.sh

```diff
@@ -0,0 +1,28 @@
+fixupOutputHooks+=(_makeSymlinksRelative)
+
+# For every symlink in $output that refers to another file in $output
+# ensure that the symlink is relative. This removes references to the output
+# has from the resulting store paths and thus the NAR files.
+_makeSymlinksRelative() {
+    local symlinkTarget
+
+    if [ -n "${dontRewriteSymlinks}" ]; then
```

Comment:
`${dontRewriteSymlinks-}` in case it is unset

Reply:
Fixed.